system/Database/SQLSRV/Connection.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+namespace CodeIgniter\Database\SQLSRV;
```

Comment:
namespace declarations should be after the file-level docblock. This place is reserved (if ever) for `declare(strict_types=1);`